website/content/api-docs/secret/openldap.mdx

```diff
@@ -119,17 +120,37 @@ $ curl \
 }
 ```
 
-## Static Role Management
+## Rotate Root Password
 
-The `static-role` endpoint configures Vault to manage the passwords of individual OpenLDAP entries.
+The `rotate-root` endpoint offers password rotation for the `binddn` entry used to manage OpenLDAP. This generated password will only be known to Vault and will not be retrievable once rotated.
 
-### Parameters
+| Method | Path                    |
+| :----- | :---------------------- |
+| `POST` | `/openldap/rotate-root` |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    http://127.0.0.1:8200/v1/openldap/rotate-root
+```
+
+## Static Roles
 
-- `dn` `(string: <required>)` - Distinguished name (DN) of entry Vault should manage. Example: `cn=bob,ou=Users,dc=hashicorp,dc=com`
-- `rotation_period` `(string: <required>)` - How often Vault should rotate the password of the user entry, in seconds. The minimum rotation period is 5 seconds. Example: "5s", "1h".
-- `username` `(string: <required>)` - The name of the user to be used when logging in. This is useful when `dn` isn't used for login purposes (such as SSH). Example: "bob".
+The `static-role` endpoint configures Vault to manage the passwords of existing individual OpenLDAP entries.
+
+### Parameters
 
-Distinguished name (DN) of entry Vault should manage. Example: `cn=bob,ou=Users,dc=hashicorp,dc=com`
+- `dn` `(string: <required>)` - Distinguished name (DN) of entry Vault should manage.<br />
+  **Example:** `cn=bob,ou=Users,dc=hashicorp,dc=com`
+- `rotation_period` `(string: <required>)` - How often Vault should rotate the password of the user entry, in seconds.
```

Comment:
Maybe revise or remove `in seconds`, especially since one of the examples is "1h".

Reply:
Wordsmithed to be similar to the `default_ttl` and `max_ttl` variables.